Source/WebCore/platform/graphics/Damage.h

```diff
@@ -25,6 +25,8 @@
 
 #pragma once
 
+#if ENABLE_WPE_PLATFORM || PLATFORM(GTK)
```

Comment:
Please use `ENABLE(WPE_PLATFORM)`:

```suggestion
#if ENABLE(WPE_PLATFORM) || PLATFORM(GTK)
```

...and also in the other occurrences in the patch.